flake8 errors in master

```
$ flake8
./build/lib/server_util/test/context.py:6:1: F401 'arkouda' imported but unused
./build/lib/server_util/test/context.py:6:1: E402 module level import not at top of file
./build/lib/server_util/test/server_test_util.py:36:1: E402 module level import not at top of file
./build/lib/server_util/test/server_test_util.py:126:9: F841 local variable 'e' is assigned to but never used
./build/lib/server_util/test/server_test_util.py:177:9: F841 local variable 'e' is assigned to but never used
./build/lib/server_util/test/server_test_util.py:289:5: F841 local variable 'e' is assigned to but never used
./build/lib/server_util/test/server_test_util.py:344:5: F841 local variable 'e' is assigned to but never used


```

--- a/setup.cfg
+++ b/setup.cfg
@@ -25,3 +25,4 @@ exclude =
     benchmarks
     pydoc/_ext/generic_linkcode_resolve_for_sphinx.py
     converter/csv2hdf.py
+    ./build

